src/data/DataProvider.ts

```diff
@@ -594,15 +596,14 @@ export class DataProvider {
   }
 
   private async _processPayments(
-    payments: ServerApi.CollectionPage<
-      | ServerApi.PaymentOperationRecord
-      | ServerApi.CreateAccountOperationRecord
-      | ServerApi.PathPaymentOperationRecord
-    >,
+    // using any type as a temp fix
+    payments: any,
```

Comment:
There are more types that we're expecting for payments that are not easy to handle down the road. Since this library is pretty much deprecated, using `any` in this case shouldn't be too bad.